rootfs/api/views.py

```diff
@@ -86,11 +87,14 @@ def destroy(self, request, **kwargs):
 
         # A user can not be removed without apps changing ownership first
         if len(models.App.objects.filter(owner=target_obj)) > 0:
-            msg = '{} still has applications assigned. Delete or transfer ownership'.format(str(target_obj))  # noqa
+            msg = '{} still has applications assigned. Please delete or transfer ownership'.format(str(target_obj))  # noqa
```

Comment:
this would mean the controller sdk also needs to be changed.
https://github.com/deis/controller-sdk-go/blob/master/errors.go#L32
https://github.com/deis/controller-sdk-go/blob/master/errors.go#L209